src/baguetteBox.js

```diff
@@ -316,6 +325,8 @@
             options.animation === 'slideIn' ? '' : 'none');
         // Hide buttons if necessary
         if (options.buttons === 'auto' && ('ontouchstart' in window || imagesMap[currentGallery].length === 1)) {
+            previousButtonContent.setAttribute('aria-hidden', 'true');
+            nextButtonContent.setAttribute('aria-hidden', 'true');
             options.buttons = false;
         }
         // Set buttons style to hide or display them
```

Comment:
I agree.

But shouldn't we have `aria-hidden=true` by default and `aria-hidden=false` if the buttons are not hidden?